tests/tests.zig

```diff
@@ -0,0 +1,14 @@
+test {
```

Comment:
This can be `comptime {` then the count of tests displayed when running `zig build test` is not increased by one.